mlrun/api/crud/model_endpoints.py

```diff
@@ -356,6 +361,115 @@ def get_endpoint(
 
         return endpoint
 
+    @staticmethod
+    def deploy_monitoring_functions(
+        project: str,
+        model_monitoring_access_key: str,
+        db_session,
+        auth_info: mlrun.api.schemas.AuthInfo,
+    ):
+        ModelEndpoints.deploy_model_monitoring_stream_processing(
+            project=project,
+            model_monitoring_access_key=model_monitoring_access_key,
+            auto_info=auth_info,
+        )
+        ModelEndpoints.deploy_model_monitoring_batch_processing(
+            project=project,
+            model_monitoring_access_key=model_monitoring_access_key,
+            db_session=db_session,
+            auth_info=auth_info,
+        )
+
+    @staticmethod
+    def deploy_model_monitoring_stream_processing(
+        project: str,
+        model_monitoring_access_key: str,
+        auto_info: mlrun.api.schemas.AuthInfo,
+    ):
+        try:
+            logger.info(
+                f"Checking deployment status for model-monitoring-stream [{project}]"
+            )
+            get_nuclio_deploy_status(
+                name="model-monitoring-stream", project=project, tag=""
+            )
+            logger.info(
+                f"Detected model-monitoring-stream [{project}] already deployed"
+            )
+            return
+        except DeployError:
+            pass
+
+        logger.info(f"Deploying model-monitoring-stream [{project}]")
+
+        fn = get_model_monitoring_stream_processing_function(project)
+        fn.metadata.project = project
+
+        stream_path = config.model_endpoint_monitoring.store_prefixes.default.format(
+            project=project, kind="stream"
+        )
+
+        fn.add_v3io_stream_trigger(
+            stream_path=stream_path, name="monitoring_stream_trigger"
+        )
+        # TODO remove custom image
+        fn.spec.image = "mlrun/mlrun:automation"
+
+        fn.set_env("MODEL_MONITORING_ACCESS_KEY", model_monitoring_access_key)
+        fn.set_env("MODEL_MONITORING_PARAMETERS", json.dumps({"project": project}))
+
+        fn.apply(mlrun.mount_v3io())
+        deploy_nuclio_function(fn, auth_info=auto_info)
+
+    @staticmethod
+    def deploy_model_monitoring_batch_processing(
+        project: str,
+        model_monitoring_access_key: str,
+        db_session,
+        auth_info: mlrun.api.schemas.AuthInfo,
+    ):
+        # Test if mlrun-job already deployed
+        function_list = get_db().list_functions(
+            session=db_session, name="model-monitoring-batch", project=project
+        )
+
+        if function_list:
+            logger.info(f"Detected model-monitoring-batch [{project}] already deployed")
```

Comment:
Same for rest of the logs here, "model monitoring batch" is a nick name that I'm not sure is so obvious
why not giving the full description model monitoring batch processing function